### DISCUSSION TITLE
Replace Komgas login page fully with Keycloaks?

### DISCUSSION THREAD

**Discussion by Schmagz** [gotson/komga#1691] [Q&A] [answered]
I've used the 'Advanved configuration' config on help page 'Social login' to setup Keycloak on Komga.
It works so I get the 'sign in with keycloak' button at login page but I was hoping to be able to replace the entire login page with Keycloaks login page so I can enable 2FA. Is this possible?

Thanks in advance

**Comment by gotson** [comment, marked as answer]
no, not possible and not planned
